server/admin/member.js

```diff
@@ -107,5 +128,9 @@ module.exports = {
     memberLeave: {
         endpoint: '/admin/member/leave',
         handler: createLeaveHandler
+    },
+    reap: {
+        endpoint: '/admin/reap',
+        handler: createReapHandler
```

Comment:
If the user calls `reap` on non-fauty node, the callback should return an error instead of OK.